src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/icon-links.html

```diff
@@ -16,9 +16,9 @@
             <span><i class="{{ icon }} fa-lg" aria-hidden="true"></i></span>
             <span class="sr-only">{{ name }}</span>
             {%- elif type == "local" -%}
-            <img src="{{ pathto(icon, 1) }}" class="icon-link-image" alt="{{ name }}"/>
+            <span><img src="{{ pathto(icon, 1) }}" class="icon-link-image" alt="{{ name }}"/></span>
```

Comment:
The span is needed for the line-height property, which was the easiest way I found to get the underline to appear in a consistent place across all header nav items (text and icon links). 

I suppose I should copy paste the following comment wherever the span is used?

```python
{# Use <span> for line-height #}
```